build/templates/__init__.py.mako

```diff
@@ -14,7 +17,22 @@ from ${module_name}.enums import *          # noqa: F403,F401,H303
 % endif
 from ${module_name}.errors import DriverWarning   # noqa: F401
 from ${module_name}.errors import Error     # noqa: F401
+<%
+# nitclk is different. It does not have a session class that we open a session on
+# Instead it is a bunch of stateless function calls. So if we are NOT building for
+# nitclk, we import the Session class like before. If it is nitclk then we will
+# import each function and the SessionReference class
+%>\
+% if config['module_name'] != 'nitclk':
 from ${module_name}.session import Session  # noqa: F401
+% else:
+from ${module_name}.session import SessionReference  # noqa: F401
+
+# Function imports
+%   for func_name in sorted([functions[k]['python_name'] for k in functions if not functions[k]['render_in_session_base']]):
```

Comment:
Added this comment:
``` python
# There are two types of functions in nitclk: functions that take a single sessions (get/set attribute) and 
# functions that take a list of sessions. The functions that take a list of sessions are considered the main
# API for this and these are the ones we need to explicitly import. We can tell which type of function each
# one is based on 'render_in_session_base'. When this is True, it means it is one of the get/set attribute
# funtions, so we skip it.
```

Reply:
Ok, sounds like you are reusing `render_in_session_base` to mean "methods in SessionReference" and the rest of the functions (the ones not `render_in_session_base`) are to become functions under `nitclk`.

Allow me to suggest clearer wording based on my understanding, since this is "clever".

```
There two types of functions in `nitclk`:

1. Functions that take a single SessionReference (get/set attribute)
2. Functions that take in a list of SessionReference

The second type are the public functions that clients will call, so we need to import them explicitly into the `nitclk` namespace. We are using the `render_in_session_base` metadata in order to distinguish them
```

Reply:
Changed